automobile/data/ir_actions_server.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
+    <record id="update_the_warranty__f6563e1c-89cd-4350-b2c5-0558e726fded" model="ir.actions.server">
+        <field name="name">update the Warranty Date</field>
```

Comment:
@kir-odoo @dhrs-odoo @jva-odoo The `x_studio_warranty_date` is defined as a related of `lot_id.expiration_date` so what is the use of that automated action/server action ?

Also could you explain a bit more flow regarding the warranty ? Usually the warranty start when selling/shipping goods to customers - here you seems to be using `expiration_date` of the `product_expiry` module which are when receiving goods

Reply:
It's try to use same field for tracking warranty period.
Tracking by serial number so same lot is not available in two locations.
It's use this custom fields because lot expiration is not visible in outgoing.

Reply:
Ok, why is `x_studio_warranty_date` field not shown in a view then ? For me it seems both `x_studio_warranty_date` and its automated action is useless

Reply:
it's there https://github.com/odoo/industry/pull/10/files#diff-118e3cecf8cd73a0873f81e511c150fcfd93a79a5bee222250eeabb86d3604d9R144